CONFIGURATION.md

```diff
@@ -1,9 +1,19 @@
 # Python Language Server Configuration
 This server can be configured using `workspace/didChangeConfiguration` method. Each configuration option is described below:
 
-| **Configuration Key** | **Type** | **Description** | **Default** 
+| **Configuration Key** | **Type** | **Description** | **Default**
```

Comment:
Was this space added in manually? The CI will complain if this file is not exactly as generated by the script.

Reply:
This was probably my editor removing leading spaces. I've regenerated the file with the script.